addons/website/tests/test_ui.py

```diff
@@ -101,9 +101,12 @@ def test_html_editor_scss(self):
 @odoo.tests.tagged('-at_install', 'post_install')
 class TestUiTranslate(odoo.tests.HttpCase):
     def test_admin_tour_rte_translator(self):
-        fr_BE = self.env.ref('base.lang_fr_BE')
-        fr_BE.active = True
-        self.env.ref('website.default_website').language_ids |= fr_BE
+        self.env['res.lang'].create({
+            'name': 'Parseltongue',
+            'code': 'pa_PA',
```

Comment:
Parseltongue from Paraguay? Maybe 
```suggestion
            'code': 'pa_GB',
```

Reply:
Makes more sense, even if I am not too sure where this language really comes from :grimacing: 
Changed :+1: